Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -901,9 +901,14 @@ def test_watcher_implementation(self):
     class SubprocessThreadedWatcherTests(SubprocessWatcherMixin,
                                          test_utils.TestCase):
         def setUp(self):
+            self._original_can_use_pidfd = unix_events.can_use_pidfd
             # Force the use of the threaded child watcher
             unix_events.can_use_pidfd = mock.Mock(return_value=False)
             super().setUp()
+        
```

Comment:
```suggestion

```

Reply:
I merged this from my phone and I didn't see the comment. Apologies!

Reply:
Don't worry I will make a PR fixing this in the next hour

Reply:
I've made https://github.com/python/cpython/pull/129180.